liquibase-integration-tests/src/test/groovy/liquibase/diffchangelog/DiffChangelogIntegrationTest.groovy

```diff
@@ -274,4 +274,36 @@ COMMENT ON COLUMN $viewName.$columnName IS '$columnComment';
         commandScope.addArgumentValue(ReferenceDbUrlConnectionCommandStep.REFERENCE_DATABASE_ARG, referenceDatabase)
         commandScope.execute()
     }
+
+    def "Treat 'null' / 'NULL' values not as NULLs"() {
+        given:
+        def outputChangelogFile = "test/classes/test-null.postgres.sql"
```

Comment:
Let's avoid outputting a changelog file in SQL format until we fix #6396 (I have started working on that issue), but for now I would try using XML or any other format.

Reply:
My main motivation is changelog in SQL format.  I can wait for #6396.

Reply:
You can ignore my previous comment because I've been trying to reproduce that issue and it seems to be working just fine now.